decimal_test.go

```diff
@@ -4040,6 +4209,45 @@ func FuzzDecimal_Mul(f *testing.F) {
 	)
 }
 
+func FuzzDecimal_Mul_Prod(f *testing.F) {
+	for _, d := range corpus {
+		for _, e := range corpus {
+			f.Add(d.neg, d.scale, d.coef, e.neg, e.scale, e.coef)
+		}
+	}
+
+	f.Fuzz(
+		func(t *testing.T, dneg bool, dscale int, dcoef uint64, eneg bool, escale int, ecoef uint64) {
+			d, err := newSafe(dneg, fint(dcoef), dscale)
+			if err != nil {
+				t.Skip()
```

Comment:
Why skipping?

Add a comment if it's normal, otherwise please fix it

It it's something you consider as a prerequisites condition before testing something else, it should fail